src/frontend/shortlist/src/components/layout/signup_login/__tests__/LoginSignup.test.js

```diff
@@ -0,0 +1,317 @@
+    const componentWrapper = mount(Login, {
+      props: {},
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const loginForm = componentWrapper.find("#loginForm");
+    expect(loginForm.exists()).toBe(true);
+    await loginForm.trigger("submit.prevent");
+  });
+
+  it("Check if the loginThroughSSO works", async () => {
+    const componentWrapper = mount(Login, {
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const loginWithGoogle = componentWrapper.find("#loginWithGoogle");
+    expect(loginWithGoogle.exists()).toBe(true);
+    await loginWithGoogle.trigger("submit.prevent");
+  });
+
+  it("Check if the switchToSignupButton works", async () => {
+    const componentWrapper = mount(Login, {
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const switchToSignupButton = componentWrapper.find("#switchToSignupButton");
+    expect(switchToSignupButton.exists()).toBe(true);
+    await switchToSignupButton.trigger("click");
+  });
+
+  it("Check if the small titles work", async () => {
+    const componentWrapper = mount(Login, {
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const smallTitles = componentWrapper.find("#small");
+    expect(smallTitles.exists()).toBe(true);
+  });
+
+  it("Check if the big titles work", async () => {
+    const componentWrapper = mount(Login, {
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const bigTitles = componentWrapper.find("#big");
+    expect(bigTitles.exists()).toBe(true);
+  });
+});
+
+describe("Signup.vue", () => {
+  it("renders with null props", () => {
+    const componentWrapper = mount(Signup, {
+      props: {
+        /*
+          TODO(Need to add API tests here)
+        */
+      },
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    // exists
+    expect(componentWrapper.exists()).toBe(true);
+  });
+
+  // The following are the component tests, all passing
+  it("Check if the signup_components_container exists", async () => {
+    const componentWrapper = mount(Signup, {
+      props: {},
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const signup_components_container = componentWrapper.find(
+      ".signup_components_container"
+    );
+    expect(signup_components_container.exists()).toBe(true);
+  });
+
+  it("Check if the signupForm works", async () => {
+    const componentWrapper = mount(Signup, {
+      props: {},
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const signupForm = componentWrapper.find("#signupForm");
+    expect(signupForm.exists()).toBe(true);
+    await signupForm.trigger("submit.prevent");
```

Comment:
Same issue, so button exist, you press the button, then ... ?

Reply:
See above